README.md

```diff
@@ -216,50 +251,118 @@ Example usage:
+were stored as integers and mapped to their string values as part of the deserialization process. After migrating to
+JSON, new enums are correctly stored as their mapped string values, but legacy data is still stored in the database
+as integers.
 
-### Database Seeds
+After consulting with stakeholders, we decided to leave the legacy data as is and map the enums in the UI,
+so as to avoid altering the audit trail. This only affects the hold status field in the Hold table, as that is the only
+enum field under `paper_trail` that we render in a view (see `app/views/hold_status`).
 
-These should only be loaded when the application database is initially set up (e.g. for new PR/development deploys or if the staging database needs to be destroyed and recreated). These seeds contain default values for certain tables such as copyrights, licenses, hold sources, and degree types.
+## Data loading
 
-The above seed data is loaded automatically during PR builds from Github. During local development it can be loaded during first deployment by running `rails db:seed`.
+There are three types of data that get loaded into this system.
 
-Additionally, degrees and departments can be manually seeded from a CSV file if desired by running `rails db:seed_degrees <csv_file_url>` and `rails db:seed_departments <csv_file_url>`, respectively. See Jira project documentation for link to a Google doc with the initial list of departments and degrees that were loaded into the production database (not maintained).
+### Database seeds
 
-Seed data is not maintained to match the production database values, which can be changed by admin users as needed. The production database *shouldn't* ever need to be reseeded.
+These should only be loaded when the application database is initially set up (e.g. for new PR/development deploys or if
+the staging database needs to be destroyed and recreated). These seeds contain default values for certain tables such as
+copyrights, licenses, hold sources, and degree types.
 
-### QA/Stakeholder Testing Data
+The above seed data is loaded automatically during PR builds from Github. During local development it can be loaded
+during first deployment by running `rails db:seed`.
 
-We're working on a process to load test data for stakeholder testing/QA in an automated fashion. Note this is different from fixture data used for automated tests. Check back soon for more info!
+Additionally, degrees and departments can be manually seeded from a CSV file if desired by running
+`rails db:seed_degrees <csv_file_url>` and `rails db:seed_departments <csv_file_url>`, respectively. See Jira project
+documentation for link to a Google doc with the initial list of departments and degrees that were loaded into the
+production database (not maintained).
 
-### Registrar Data
+Seed data is not maintained to match the production database values, which can be changed by admin users as needed. The
+production database *shouldn't* ever need to be reseeded.
```

Comment:
I think we should state this final sentence even more strongly as "Do not ever reseed the production database".